syntax = "proto3";

package com.example;

message RandomNumberConfig {}

message RandomNumberRequest {
  int32 range_start = 1;
  int32 range_end = 2;
}
message RandomNumberResponse {
  int32 result = 1;
}

service RandomNumberService {
  // Get a random number in the range of range_start and range_end.
  rpc GetRandomNumber(RandomNumberRequest)
      returns (RandomNumberResponse) {}
}
